jobs/nfsbroker/spec

```diff
@@ -47,3 +47,20 @@ properties:
   nfsbroker.db_ca_cert:
     default: ""
     description: "(optional) CA Cert to verify SSL connection, if not include, connection will be plain"
+  nfsbroker.allowed-in-source:
+    description: "This is a comma separted list of parameters allowed to be send in share url. Each of this parameters can be specify by brokers"
```

Comment:
Wording/typos in descriptions. Please consider descriptions to match proposal on https://github.com/cloudfoundry-incubator/nfsv3driver/issues/2#issuecomment-276995586